src/frontend/classic/components/ThumbnailViewer.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+    import type { IMediaContainer } from "../../../engine/providers/MediaPlugin";
+    import ThumbnailViewerImage from "./ThumbnailViewerImage.svelte";
+
+    export let item: IMediaContainer;
+    export let throttlingDelay: number;
+    export let toggleWideViewer: (currentImage: number) => void;
+</script>
+
+<div>
+    {#each item.Entries as content, index}
+        <ThumbnailViewerImage
+            src={content.SourceURL}
+            throttlingDelay={throttlingDelay * index + 1}
```

Comment:
Throttling is not a delay, it's all about resolving the promise of the SourceURL.
but... i'll handle this myself, leave it like that to test animation/Transitions.